### DISCUSSION TITLE
Student Pack Help!

### DISCUSSION THREAD

**Discussion by zendx14** [community/community#42047] [GitHub Education]
I recently joined a school in my area and came to know that GitHub provides student benefits to young age. I am 16 and got a declaration form from my school body stating about my admission. When i sign up for GitHub student pack, while uploading docs it directly opens my camera whereas i have a pdf received from my school. How do I directly upload the pdf

**Comment by elli0t43** [comment]
> I recently joined a school in my area and came to know that GitHub provides student benefits to young age. I am 16 and got a declaration form from my school body stating about my admission. When i sign up for GitHub student pack, while uploading docs it directly opens my camera whereas i have a pdf received from my school. How do I directly upload the pdf

Can you be a bit more specific which device you are on ? 
If you are on PC you can simply disable the camera toggle from site settings for that specific web page

**Comment by getlost01** [comment]
To upload a PDF file to GitHub when signing up for the GitHub Student Developer Pack, you can follow these steps:

1. Navigate to the GitHub Student Developer Pack page (https://education.github.com/pack).
2. Click on the "Get the Pack" button.
3. On the next page, click on the "Sign up for GitHub" button.
4. Follow the prompts to create a new GitHub account.
5. When you reach the page where you are asked to provide verification of your student status, click on the "Choose a file" button.
6. This will open a file picker that allows you to select a file from your computer.
7. Navigate to the location where you have saved the PDF file received from your school, and select it.
8. Click on the "Open" button to attach the file to the form.
9. Review the form and make sure all the required fields are filled out correctly.
10. Click on the "Submit" button to complete the sign-up process.

If you are having trouble uploading the PDF file, make sure that the file is in the correct format and that it is not corrupted. You may also want to try using a different browser or clearing your cache and cookies to see if that helps.

**Comment by zendx14** [comment]
I am using a mobile, i do not see any option to upload, rather it asks to
take a photo
